tools/internal_ci/macos/grpc_run_bazel_isolated_tests.sh

```diff
@@ -26,6 +26,8 @@ tools/bazel test $RUN_TESTS_FLAGS --spawn_strategy=local --genrule_strategy=loca
 
 # Make sure time is in sync before running time_jump_test because the test does
 # NTP sync before exiting. Bazel gets confused if test end time < start time.
+sudo touch /var/db/ntp-kod
```

Comment:
add explanation why this is needed and the b/ short link (otherwise no one is going to remember why this is important).

also add a TODO run investigate how to run time_jump_test without needing to mess with the system time directly (which makes the test difficult to run, to reproduce issues with it and it also requires it to live in a dedicated kokoro job - all of which is hassle).

Reply:
Done, thanks for your suggestion!